app/models/child.rb

```diff
@@ -114,6 +114,14 @@ def hot_lunch_dates
     end
   end
 
+  def completed_envelope?
+    childcare_envelopes.map(&:status).include?(ChildcareEnvelope::COMPLETED_ENVELOPE_STATUS)
```

Comment:
Would using `childcare_envelopes.pluck(:status)` be a bit more efficient?

Reply:
👍